via/exceptions.py

```diff
@@ -1,24 +1,10 @@
 """Application specific exceptions."""
 
 from pyramid.httpexceptions import HTTPBadRequest, HTTPConflict, HTTPExpectationFailed
-from requests import exceptions
 
 # pylint: disable=too-many-ancestors
 # It's ok to have a hierarchy of exceptions
 
-REQUESTS_BAD_URL = (
-    exceptions.MissingSchema,
-    exceptions.InvalidSchema,
-    exceptions.InvalidURL,
-    exceptions.URLRequired,
-)
-REQUESTS_UPSTREAM_SERVICE = (
-    exceptions.ConnectionError,
-    exceptions.Timeout,
-    exceptions.TooManyRedirects,
-    exceptions.SSLError,
-)
```

Comment:
It doesn't seem like this was the right place for these, to me. `exceptions.py` is where we keep our custom exception classes and exception views. These are just some constants for grouping together different types of `requests` exceptions. I think the constants probably should have always been defined where they were used (and they don't seem like they should be used in more than one place if we have the code design right)

Reply:
That's the situation we now have, which is better.